README.md

```diff
@@ -12,17 +12,23 @@ Use your text editor to write in your browser. Everything you type in the editor
     + [**Sublime Text** extension](https://sublime.wbond.net/packages/GhostText) - [Repo](https://github.com/GhostText/GhostText-for-SublimeText)
     + [**Atom** package](https://github.com/GhostText/GhostText-for-Atom)
     + [**VS Code** extension](https://marketplace.visualstudio.com/items?itemName=tokoph.ghosttext) - [Repo](https://github.com/jtokoph/ghosttext-vscode) (Third party)
-    + [**Vim** script](https://github.com/falstro/ghost-text-vim) (Third party)
-    + [**Vim** plugin](https://github.com/pandysong/ghost-text.vim) (Third party)
-    + [**Neovim** plugin](https://github.com/raghur/vim-ghost) (Third party)
     + [**Emacs** package](https://melpa.org/#/atomic-chrome) - [Repo](https://github.com/alpha22jp/atomic-chrome) (Third party)
     + [**Acme** client](https://github.com/fhs/Ghost) (Third party)
+    + <details>
+        <summary><b>Vim</b>/<b>Neovim</b> (Third party)</summary>
+        <ul>
+          <li><a href="https://github.com/falstro/ghost-text-vim">Tcl script for Vim</a>
+          <li><a href="https://github.com/pandysong/ghost-text.vim">Python plugin for Vim</a>
+          <li><a href="https://github.com/raghur/vim-ghost">Python plugin for Vim/Neovim</a>
+          <li><a href="https://github.com/subnut/nvim-ghost.nvim">Native Neovim plugin</a>
```

Comment:
Are these descriptions correct? Does "native" make sense for a neovim plugin written in vim?

Reply:
This is even clearer and follows the others (they all start with the editor’s name)

```suggestion
          <li><a href="https://github.com/falstro/ghost-text-vim">Vim (written in Tcl)</a>
          <li><a href="https://github.com/pandysong/ghost-text.vim">Vim (written in Python)</a>
          <li><a href="https://github.com/raghur/vim-ghost">Vim/Neovim (written in Python) </a>
          <li><a href="https://github.com/subnut/nvim-ghost.nvim">Neovim (written in Vim)</a>
```